packages/email-plugin/src/types.ts

```diff
@@ -31,6 +31,41 @@ export type EventWithContext = VendureEvent & { ctx: RequestContext };
  */
 export type EventWithAsyncData<Event extends EventWithContext, R> = Event & { data: R };
 
+/**
+ * @description
+ * Allows you to dynamically load the "globalTemplateVars" key async and access Vendure services
+ * to create the object. This is not a requirement. You can also specify a simple static object if your
+ * projects doesn't need to access async or dynamic values.
+ *
+ * @example
+ * ```ts
+ *
+ * EmailPlugin.init({
+ *    globalTemplateVars: async (ctx, injector) => {
+ *          const myAsyncService = injector.get(MyAsyncService);
+ *          const asyncValue = await myAsyncService.get(ctx);
+ *          const channel = ctx.channel;
+ *          const { primaryColor } = channel.customFields.theme;
+ *          const theme = {
+ *              primaryColor,
+ *              asyncValue,
+ *          };
+ *          return theme;
+ *      }
+ *   [...]
+ * })
+ *
+ * ```
+ *
+ * @docsCategory core plugins/EmailPlugin
+ * @docsPage EmailPluginOptions
+ * @docsWeight 0
```

Comment:
Setting `docsWeight 0` will force this type to the top of the `EmailPluginOptions` docs page, which is not what we want. Better to have the main `EmailPluginOptions` interface at the top. So remove this to be safe add it to the `EmailPluginOptions` doc block to ensure it always stays at the top.